nifi-registry/nifi-registry-core/nifi-registry-data-model/pom.xml

```
@@ -33,6 +33,11 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-api</artifactId>
+            <version>1.14.0-SNAPSHOT</version>
```

This should be updated to 1.15.0-SNAPSHOT to match the current main branch.
```suggestion
            <version>1.15.0-SNAPSHOT</version>
```